databrowser/src/domain/homepage/utils.ts

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: NOI Techpark <digital@noi.bz.it>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import { LocationQueryValue, Router } from 'vue-router';
+import { useRouteQuery } from '@vueuse/router';
+
+import * as R from 'ramda';
+
+export function useUpdateURL(router: Router, filters: string[] = [], searchQuery: string = '') {
+  const currentQuery = router.currentRoute.value.query;
+  const newQuery: { [key: string]: string | LocationQueryValue[] | null } = {
+    ...currentQuery,
+  };
+  if (!filters || filters.length > 0) {
```

Comment:
What does the `!filter` check? `filter` is typed as array, so a length check should suffice